### PR TITLE
Bump to 2.31.6

### PR DESCRIPTION
**Pull request by PettitWesley**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: windows.versions [hunk 1]
```diff
@@ -6,12 +6,24 @@
       "fluent-bit": "1.9.10",
       "kinesis-plugin": "v1.10.1",
       "firehose-plugin": "v1.7.1",
-      "cloudwatch-plugin": "v1.9.1",
+      "cloudwatch-plugin": "v1.9.2",
```
Comment (ShelbyZ) [inline windows.versions:9]: We also need to bump the version above to 2.31.6 :)

**Review by ShelbyZ — CHANGES_REQUESTED** [review]
Only need the version bump and we are good to go

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+### 2.31.6
```
[the remark this answers is not in the document]
Comment (PettitWesley) [reply CHANGELOG.md:3]: Yea I like it, it makes the release notes way shorter

**Review by ShelbyZ — APPROVED** [review]

**Review by matthewfala — APPROVED** [review]
Looks good. I like the deep link idea.

**Pull request merged**